contrib/question_converters/randexam/randexam2pl2.py

```diff
@@ -189,10 +189,11 @@ def __init__(self):
 
 
 def ind2chr(index):
-    """c = ind2chr(i)
-
-    Convert the index i to a character c, so that 0 -> 'A', 1 -> 'B',
+    """Convert the index i to a character c, so that 0 -> 'A', 1 -> 'B',
     etc. Invalid indexes convert to the character '*'.
+
+    Examples:
+        >>> c = ind2chr(i)
```

Comment:
Example isn't really instructive here

Reply:
Improved in [533ae66](https://github.com/PrairieLearn/PrairieLearn/pull/11758/commits/533ae668d695d5ca5a3f241cdcd98776a7728dbf).